dbms/src/test/java/org/polypheny/db/jdbc/JdbcDdlTest.java

```diff
@@ -652,7 +649,7 @@ public void addColumnArrayTest() throws SQLException {
 
 
     @Test
-    @Category({ FileExcluded.class })
+    @Tag("fileExcluded") // we have to add convert methods for all PolyValues, then we can enable
```

Comment:
Can you enable this test?

Reply:
No, this requires quite some additional logic, just to make this work. Should do that in a future PR.